VERSION

```diff
@@ -1 +1 @@
-1.31.8
\ No newline at end of file
+1.32.0
```

Comment:
Yup, I'm OK with that. :) Gives me something to talk to in the upcoming release announcement for people to understand that patch version numbers will likely be higher from now on.